api/proto-go/core/pipeline/base/pb/base.pb.go

```diff
@@ -4359,6 +4359,10 @@ type PipelineCreateRequest struct {
 	// +optional
 	DefinitionID string           `protobuf:"bytes,18,opt,name=definitionID,proto3" json:"definitionID,omitempty"`
 	IdentityInfo *pb.IdentityInfo `protobuf:"bytes,19,opt,name=identityInfo,json=-,proto3" json:"identityInfo,omitempty"`
+	// Secrets Environment variables passed from the upper layer，different from configs cms
```

Comment:
Secrets variables, not env.

Reply:
passed from the invoker, different from config cms

Reply:
passed from the invoker, different from config cms

Reply:
done